pkgs/tools/inputmethods/ibus/wrapper.nix

```diff
@@ -7,7 +7,6 @@ let
   env = {
     buildInputs = [ ibus ] ++ plugins;
     nativeBuildInputs = [ lndir makeWrapper ];
-    propagatedUserEnvPackages = [ hicolor_icon_theme ];
```

Comment:
IIRC hicolor is used for systray icons, not sure about what `propagatedUserEnvPackages ` is and what it is used for though (KDE integration?).
@ttuegel should give you a better explanation of this code.